topics/evolution/tutorials/abc_intro_phylo/tutorial.md

```diff
@@ -0,0 +1,1293 @@
+
+> <hands-on-title>Obtain your data</hands-on-title>
+>
+> 1. Make sure you have an empty analysis history. Give it a name.
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+> 2. Import the file anolis-raw.fst from [Zenodo](https://zenodo.org/records/10415242/files/anolis-raw.fst) or from the shared data library.
+>    ```
+>    https://zenodo.org/records/10415242/files/anolis-raw.fst
+>    ```
+>    You may need to refresh your history to show the data.
+>    If the paste/fetch link does not work for you, you can download it to your own computer and then upload it using the "Upload" button at the top of the left panel in Galaxy.
+>    
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+> 
+{: .hands_on} 
+
+You can click on the 'eye' icon {% icon galaxy-eye %} on the right to see the unaligned data (go ahead!) but the view isn't very informative. This is the raw FASTA file, with the symbols A, C, G, T representing nucleotides. You can see that the sequences are different lengths.
+
+**FASTA** or **Fasta** format (pronounced to rhyme with pasta) format is is commonly used as input to phylogenetic inference programs and has a very simple structure, as follows:
+
+1. Each sequence has a name, which appears on its own line after a '>' sign 
+2. The next line(s) contain the sequence; continuing either until the next sequence name line, or the end of the file.
+
+The example below has four sequences in it named Taxon_1, Taxon_2, etc, each with a short set of characters representing DNA sequences.
+
+
+```
+>Taxon_1
+ACTGCGTTAGGTCTAGCC
+>Taxon_2
+GATCTACTGCTTTAGGTTGAGCC
+>Taxon_3
+ACTGCTCTAGCACTGAGCCCA
+>Taxon_4
+ACTTGGCGTAGCCGGAGGCC
+```
+
+
+
+<!-- A line beginning with the greater-than sign `>` holds the sequence name or identifier/ID; the other lines hold the sequence data for the sequence ID above. -->
+
+> <tip-title>FASTA format for multiple sequence alignments</tip-title>
+>
+>The Fasta format can also include symbols such as a question mark '?' for missing data, or hyphen '-' to indicate an insertion or deletion events, collectively *indels*. It is common to represent a multiple sequence alignment in Fasta format with these symbols. It is also a common format for storing high-throughput read data, but *without quality scores* -- if you want to include read quality you would use FASTQ format.
+{: .tip}
+
+> <hands-on-title>View your data</hands-on-title>
+> Let's view the unaligned sequence in a more understandable form.
+>
+> 1. Click on the title of your file to see the row of small icons for saving, linking etc: 
+> ![Screenshot of Galaxy icons. Seven small blue icons are shown on a green background. From left to right they are: floppy disk, link, information, redo, bar chart, flow chart and a question mark.](./images/LittleJobIcons.png) 
+>
+> 2. Click on the **visualise icon** {% icon galaxy-visualise %} and then select the **Multiple Sequence Alignment tool**.
+> You should see something like this:
+>
```

Comment:
I'm experiencing an issue here (filed internally as https://github.com/usegalaxy-eu/issues/issues/522), namely that the MSA visualisation will not visualise unaligned sequences, it gives an "unknown format error".

Based on the screenshot though, y'all clearly have this working somewhere. Is that Galaxy AU? Or perhaps the dev/test sites of AU? Or a local install?

I want to be sure that we aren't telling as essentially step 2 in this tutorial to do something that generates a completely unhelpful error message (which should be fixed in the visualisation)

Reply:
Thanks @wm75 who noticed the issue, the anolis-raw.fst on zenodo includes a blank line as it's first line.

```
$ curl --silent https://zenodo.org/records/10415242/files/anolis-raw.fst | head

>Anolis.acutus
ATGAGTCCCACAATCTATACAATTATTATATCAAGCCTAGCAACAGGCACAATTATTACCATAACCAGCTACCACTGACTTATAGCTTGAATAGGCCTAGAACTTAATACACTAGCAATTATTCCAATCATTTCATCACAACACCACCCCCGATCTACAGAAGCTGCAACAAAATATTTTCTGACACAAGCAGCAGCATCCGCCATAATTTTATTCTCCAGCATATCCAATGCCTGANTTACTGGTACATGGGACATCACTCAAATATCTTACACCCCCTCTAATATTTTTTTAACCATAGCACTAGCTATAAAATTAGGTTTGGCACCAGTACACTTCTGATTTCCAGAAGTATTACAAGGATCAACCTTATTTACTGCTTTTATTATTTCAACATGACAAAAGTTAGCCCCAATATCACTAA
```

this is preventing the correct display. Could the zenodo record be updated?

Reply:
Filed upstream as https://github.com/galaxyproject/galaxy/issues/18066

Reply:
Thanks @hexylena and @wm75. I've updated the file in Zenodo and the links to Zenodo in the tutorial. It now works on Galaxy AU.